IMDA-BTG/aiverify-developer-tools
# Update doc for integrated developer tool

See https://github.com/IMDA-BTG/aiverify-developer-tools/issues/12.

To test, run mkdocs and check:
1. New doc "Creating your First Plugin" under "Guided Example" menu.
2. Go through the guided examples and make sure that the tutorial has been updated to reference the integrated ai-verify-plugin commands.
